auth.php

```diff
@@ -116,7 +116,7 @@ public function user_signup($user, $notify=true) {
         $user->picture = 0;
         $user->imagealt = 0;
         $user->deleted = 0;
-        $user->policyagreed = 0;
+        // $user->policyagreed = 0; // Agreement to policy is triggered before the signup form, this would reset it 
```

Comment:
Please don't comment out code in the patch, just either fix it to what it should be or remove the line completely.

The CI was complaining because comments must end with punctuation:

https://travis-ci.org/catalyst/moodle-auth_enrolkey/jobs/652233553#L891

Reply:
I corrected the PR (hopefully). I find all this Github stuff more than confusing...